blocks/subscriptions-block/components/useOffer.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect, useCallback } from 'react';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import { isServerSide } from '@wpmedia/engine-theme-sdk';
+
+const offerService = ({
+  origin,
+  code,
```

Comment:
I think we should rename this to `campaignCode` or `campaignName` to match the API: https://corecomponents.arcpublishing.com/alc/docs/swagger/?url=./arc-products/arc-retail.json#/Offers/getLive

Reply:
also maybe default it to default here?
```suggestion
  campaignCode = 'default',
```